libraries/core_libs/consensus/src/vote_manager/vote_manager.cpp

```diff
@@ -416,33 +365,22 @@ void VoteManager::cleanupVotes(uint64_t pbft_round) {
     }
   }
 
-  auto batch = db_->createWriteBatch();
-  for (auto const& v_hash : remove_unverified_votes_hash) {
-    db_->removeUnverifiedVoteToBatch(v_hash, batch);
-  }
-  db_->commitWriteBatch(batch);
-
   // Remove verified votes
-  std::vector<vote_hash_t> remove_verified_votes_hash;
+  auto batch = db_->createWriteBatch();
   {
     UniqueLock lock(verified_votes_access_);
     auto it = verified_votes_.begin();
     while (it != verified_votes_.end() && it->first < pbft_round) {
       for (auto const& step : it->second) {
         for (auto const& voted_value : step.second) {
           for (auto const& v : voted_value.second) {
-            remove_verified_votes_hash.emplace_back(v.first);
+            if (v.second->getVoterAddr() == node_addr_) db_->removeVerifiedVoteToBatch(v.first, batch);
```

Comment:
missing parenthesis for if

Reply:
Why only remove node own votes and not remove other peers votes?

Reply:
as I stated in `Purpose` of this PR, we are saving in DB only our own votes

Reply:
so there are no other votes to remove, that's why there is that condition